contrib/pybossa.ini.template

```diff
@@ -0,0 +1,13 @@
+[uwsgi]
+socket = /tmp/pybossa.sock
+chmod-socket = 666
+chdir = /home/pybossa/pybossa
+pythonpath = ..
+virtualenv = /home/pybossa/pybossa/env
+module = run:app
+cpu-affinity = 1
+processes = 2
+threads = 2
+listen = 2048
```

Comment:
remove that line, as people will need to modify the kernel parameters via sysctl :-)